cmu_graphics/utils.py

```diff
@@ -59,6 +59,8 @@ def polygonContainsPoint(pts, px, py):
         q1y = q1[1]
         q2x = q2[0]
         q2y = q2[1]
+        if distanceToLineSegment2(px, py, q1x, q1y, q2x, q2y) < 0.0002:
```

Comment:
What's going on on this line?

Reply:
This is a port of https://github.com/cmu-cs-academy/cs-academy/pull/2708